### PR TITLE
Ensure ProcessedFieldOrPropertyInitializers.Empty is immutable

### LINKED ISSUE #4003: Microsoft.CodeAnalysis.Scripting.VisualBasic.UnitTests.InteractiveSessionTests.AnonymousTypes_TopLevel_MultipleSubmissions test failure

### PR DESCRIPTION
**Pull request by cston**
Ensure ProcessedFieldOrPropertyInitializers.Empty is immutable

Fixes #4003

### CODE REVIEW

**Comment by cston** [comment]
@tmat, @amcasey

File: src/Compilers/VisualBasic/Portable/Binding/Binder_Initializers.vb [hunk 1]
```diff
@@ -30,12 +30,18 @@ Namespace Microsoft.CodeAnalysis.VisualBasic
                 End Get
                 Set(value As ImmutableArray(Of BoundStatement))
                     Debug.Assert(Not value.IsDefault)
-                    ImmutableInterlocked.InterlockedInitialize(_loweredInitializers, value)
+                    Debug.Assert(_loweredInitializers.IsDefault)
```
Comment (amcasey) [inline src/Compilers/VisualBasic/Portable/Binding/Binder_Initializers.vb]: I must be missing something obvious.  Isn't this set to Empty in the constructor?

Comment (cston) [inline src/Compilers/VisualBasic/Portable/Binding/Binder_Initializers.vb]: There are two constructors. This field is not set in the public constructor.

**Comment by amcasey** [comment]
:+1:

**Pull request merged**